varsim.py

```diff
@@ -21,6 +23,56 @@
 REQUIRE_VARSIMJAR = not os.path.isfile(DEFAULT_VARSIMJAR)
 if REQUIRE_VARSIMJAR: DEFAULT_VARSIMJAR = None
 
+def convertCN(filenames, operation):
+    """
+    convert '2/1'-like copy number to a single number(e.g. 2)
+    0 will be considered same as 1
+    by default the max number will be kept
+    the change is in place
+    """
+    if operation != "two2one" and operation != "one2two":
+        raise ValueError("Only two2one or one2two allowed")
+    two2one = operation == "two2one"
+    delimiter = re.compile('[/|]')
+    for name in filenames:
+        with open(name, 'r') as file_fd:
+            output = tempfile.NamedTemporaryFile(mode = 'r+w', delete = False)
+            for l in file_fd:
+                l = l.rstrip()
+                fields = l.split("\t")
+                if l.startswith("#") or 'CN' not in fields[8]:
+                    if l.startswith('##FORMAT=<ID=CN'):
+                        if two2one:
+                            l = l.replace("Type=String","Type=Integer")
+                        else:
+                            l = l.replace("Type=Integer", "Type=String")
+                    output.write(l + "\n")
+                else:
+                    info = fields[8].split(':')
+                    cnIndex = info.index('CN')
+                    gtIndex = info.index('GT')
+                    #change CN field in all samples
+                    for sampleIndex in range(9,len(fields)):
+                        sampleInfo = fields[sampleIndex].split(':')
+                        if two2one:
+                            cn = delimiter.split(sampleInfo[cnIndex])
+                            sampleInfo[cnIndex] = str(max(cn))
```

Comment:
Taking `max` over a string array does not seem right. Don't you need to convert `cn` to a list of ints?